modules/synthetics/main.tf

```diff
@@ -4,16 +4,39 @@ locals {
   enabled            = module.this.enabled
   alert_tags         = local.enabled && var.alert_tags != null ? format("%s%s", var.alert_tags_separator, join(var.alert_tags_separator, var.alert_tags)) : ""
   datadog_synthetics = { for k, v in var.datadog_synthetics : k => v if local.enabled }
+
+  all_public_locations = [
+    "aws:ap-northeast-1",
```

Comment:
Couldnt we retrieve this from a data source ?

Reply:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/regions

Reply:
🤦 didn't see, but maybe `data "datadog_synthetics_locations" "test" {}`